apps/charge-api-service/src/explorer-api/explorer-api.interceptor.ts

```diff
@@ -60,10 +60,11 @@ export class ExplorerApiInterceptor implements NestInterceptor {
     const requestConfig: AxiosRequestConfig = {
       url: `${config?.baseUrl}`,
       method: ctxHandlerName
+
     }
 
     if (!isEmpty(query)) {
```

Comment:
Could there be cases that `query` is empty? If query is empty, the API key wouldn't be appended. @NikolayRn

Reply:
Yes, it is ok